src/index.js

```diff
@@ -59,26 +62,40 @@ const stepTrends = $('#step-trends');
 const stepGoalChart = $('#step-goal-chart');
 const friendList = $('#friend-list');
 
-const randomId = Math.floor(Math.random() * (50 - 1) + 1);
-const userRepository = new UserRepository(userData, randomId);
-const hydrationRepository = new HydrationRepository(hydrationData, randomId);
-const sleepRepository = new SleepRepository(sleepData, randomId);
-const activityRepository = new ActivityRepository(randomId, activityData);
-const user = new User(userRepository.getUserData());
-
-updateUserDataDOM(userRepository.getUserData());
-compareStepGoal(userRepository.getUserData());
-displayDailyOz();
-displayWeeklyOz();
-displayBestSleepers();
-displayCurrentDate(getCurrentDate());
-displaySleep();
-displayActivity();
-displayAverageWeeklyActivity();
-displayWeeklyActivity();
-friendActivityData(getCurrentDate());
-displayTrends();
-displaySleepChart()
+var randomId;
+var userRepository;
```

Comment:
Is there a reason you chose var over let?  It seems like both should be manipulatable on the global scope?